common.groovy

```diff
@@ -40,3 +40,15 @@ defaults = [
     channel: '#testing',
   ],
 ]
+
+E2E_RUNNER_JOB = '''
```

Comment:
nice -- but `workflow-test(-pr)` doesn't define `RELEASE` as far as I'm aware... so we could either define it as `dev` in that job or move the two exports to only the release test job...